### ISSUE TITLE
cannot import name 'checkboxlist_dialog': need for prompt toolkit

### ISSUE THREAD

**Issue by cryptax** [quark-engine/quark-engine#186]
With Quark 21.6, I get this error. I believe you need to specify you need **prompt-toolkit 3.0**

```
$ quark 
Traceback (most recent call last):
  File "/home/axelle/.local/bin/quark", line 5, in <module>
    from quark.cli import entry_point
  File "/home/axelle/.local/lib/python3.8/site-packages/quark/cli.py", line 14, in <module>
    from quark.Objects.quark import Quark
  File "/home/axelle/.local/lib/python3.8/site-packages/quark/Objects/quark.py", line 24, in <module>
    from quark.utils.graph import call_graph
  File "/home/axelle/.local/lib/python3.8/site-packages/quark/utils/graph.py", line 9, in <module>
    from prompt_toolkit.shortcuts import checkboxlist_dialog
ImportError: cannot import name 'checkboxlist_dialog' from 'prompt_toolkit.shortcuts' (/usr/lib/python3/dist-packages/prompt_toolkit/shortcuts/__init__.py)
```

Modify `setup.py` with this to get it to work

```python
       "prompt-toolkit==3.0.0",
```

**Comment by krnick** [comment]
Hello @cryptax , thanks for letting me know about this problem!
I will modify `setup.py` and release it on PyPi soon.

**Comment by krnick** [comment]
Hi @cryptax, I have updated the version of `prompt-toolkit` on PyPi, please help me to check if it works in your environment now.
Many thanks!

**Comment by krnick** [comment]
Closed in PR #187
